<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>338</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>UDP Multicast</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>组播端口</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinPort">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>35320</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>组播地址</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboIP">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="duplicatesEnabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>239.255.43.21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>239.255.100.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>239.255.255.100</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="editMsg"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnMulticast">
        <property name="text">
         <string>组播消息</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="plainText">
       <string>239.0.0.0～239.255.255.255为本地管理组播地址范围</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
   </property>
   <property name="text">
    <string>加入组播</string>
   </property>
   <property name="toolTip">
    <string>加入组播</string>
   </property>
  </action>
  <action name="actStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
   </property>
   <property name="text">
    <string>退出组播</string>
   </property>
   <property name="toolTip">
    <string>退出组播</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本程序</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/212.bmp</normaloff>:/images/images/212.bmp</iconset>
   </property>
   <property name="text">
    <string>清空文本框</string>
   </property>
   <property name="toolTip">
    <string>清空文本框</string>
   </property>
  </action>
  <action name="actHostInfo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/110.JPG</normaloff>:/images/images/110.JPG</iconset>
   </property>
   <property name="text">
    <string>本机地址</string>
   </property>
   <property name="toolTip">
    <string>获取本机地址</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
